self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryTasks.java

```diff
@@ -129,6 +129,24 @@ public Task assign(
         return assigned;
     }
 
+    @Override
+    public Task unassign(final Task task) {
```

Comment:
@amihaiemil InMemoryTasks#unassign is implemented here :)

Reply:
@criske right, I missed it, sorry :D